src/App.vue

```diff
@@ -17,13 +22,68 @@ export default {
 }
 </script>
 
+
 <style>
```

Comment:
Can you read [this](https://vue-loader.vuejs.org/guide/scoped-css.html) documentation ?